modules/govuk_ci/manifests/master.pp

```diff
@@ -57,4 +59,9 @@
     mode => '0555',
   }
 
+  create_resources('::govuk_jenkins::ssh_slave', $ci_agents, {
+    'credentials_id' => $credentials_id,
+    'labels'         => 'mongo-3.2',
```

Comment:
This is a default label. I put it in here to try and cut down the default hiera, but it probably makes more sense to be verbose in the hiera. WIll update.